[2024-02-26T15:51:54.003+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_0 manual__2024-02-26T15:51:45+00:00 [queued]>
[2024-02-26T15:51:54.036+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_0 manual__2024-02-26T15:51:45+00:00 [queued]>
[2024-02-26T15:51:54.038+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-02-26T15:51:54.063+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): runme_0> on 2024-02-26 15:51:45+00:00
[2024-02-26T15:51:54.070+0000] {standard_task_runner.py:60} INFO - Started process 1906 to run task
[2024-02-26T15:51:54.103+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_0', 'manual__2024-02-26T15:51:45+00:00', '--job-id', '2195', '--raw', '--subdir', '/home/***/.local/lib/python3.11/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp5dhw0h6o']
[2024-02-26T15:51:54.108+0000] {standard_task_runner.py:88} INFO - Job 2195: Subtask runme_0
[2024-02-26T15:51:54.374+0000] {task_command.py:423} INFO - Running <TaskInstance: example_bash_operator.runme_0 manual__2024-02-26T15:51:45+00:00 [running]> on host d07e1489670f
[2024-02-26T15:51:54.654+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_0' AIRFLOW_CTX_EXECUTION_DATE='2024-02-26T15:51:45+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-26T15:51:45+00:00'
[2024-02-26T15:51:54.657+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-26T15:51:54.662+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "example_bash_operator__runme_0__20240226" && sleep 1']
[2024-02-26T15:51:54.721+0000] {subprocess.py:86} INFO - Output:
[2024-02-26T15:51:54.727+0000] {subprocess.py:93} INFO - example_bash_operator__runme_0__20240226
[2024-02-26T15:51:55.732+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-26T15:51:55.764+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20240226T155145, start_date=20240226T155154, end_date=20240226T155155
[2024-02-26T15:51:55.817+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-26T15:51:55.845+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
